sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/RestrictDeserializedTypes.LosFormatter.cs

```diff
@@ -8,6 +8,7 @@ internal class RestrictDeserializedTypes
         public void DefaultConstructor()
         {
             new LosFormatter(); // Noncompliant {{Serialized data signature (MAC) should be verified.}}
+            new LosFormatter { }; // Noncompliant
```

Comment:
it would be nice to search in our code base for similar patterns , to add it to other unit tests (unrelated to this PR)